openpype/hosts/nuke/plugins/publish/extract_review_data_mov.py

```diff
@@ -58,7 +72,7 @@ def process(self, instance):
                     continue
 
                 # test task types from filter
-                if f_task_types and not any(
+                if f_task_types and not bool(
                         task_type in f_task_types):
```

Comment:
There's no need to turn this into a `bool` explicitly.

```suggestion
                if f_task_types and task_type not in f_task_types:
```